RecoEgamma/EgammaTools/plugins/EGRegressionModifier.h

```diff
@@ -1,27 +1,14 @@
-#include "CommonTools/CandAlgos/interface/ModifyObjectValueBase.h"
+#ifndef RecoEgamma_EgammaTools_EGRegressionModifier_H
+#define RecoEgamma_EgammaTools_EGRegressionModifier_H
```

Comment:
I'm missing the purpose of this file.
- it's named as if it's meant to give a base class for EGRegressionModifierV1 and EGRegressionModifierV2 (but it does not)
- it declares a helper method retrieveGBRForests, which ends up being implemented only in EGRegressionModifierV2, but it's also used in EGRegressionModifierV1, which is rather awkward.

Please either rename this to something more descriptive like EGRegressionModifierHelpers and implement the method in .cc or actually complete the inheritance reorganization or EGRegressionModifierV[12]